Source/swiftlint/Helpers/LintableFilesVisitor.swift

```diff
@@ -28,6 +28,16 @@ enum LintOrAnalyzeModeWithCompilerArguments {
     case analyze(allCompilerInvocations: CompilerInvocations)
 }
 
+private func resolveParamsFiles(args: [String]) -> [String] {
```

Comment:
I named this more generically since theoretically this could be used for things besides paths